config.yaml

```diff
@@ -43,10 +43,7 @@ manager:
   # A path to a Cloudify license to be uploaded on the Manager during installation
   cloudify_license_path: ''
 
-cluster:
-  # When running a cluster, every extra node must contact any active manager
-  # in the cluster, this will not work on non-premium installations
-  active_manager_ip: ''
+cluster: {}
```

Comment:
Probably better to change the `config[CLUSTER]['join']` to `config[CLUSTER_JOIN]` so we don't need this stale key in the `config.yaml`

Reply:
maybe, i just expect we will have _some_ cluster configuration soon, you think drop until we need it? keeping it means we won't have to change the join again

Reply:
btw the join entirely is somewhat of a hack because it's just a way to pass info from db.py to cluster.py (which is a different component)
we dont have another way to do that than config but i wish we did :/

Reply:
Yeah...
Anyway I don't aee any cluster configurations we're going to need tbh eapecially not in 5.0, the only reason we ever had it was to point to the active/master manager...